lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -87,6 +87,11 @@
     required: false
     version_added: "2.1"
     type: bool
+  network:
+    description:
+      - The network to use for RUN build instructions.
```

Comment:
You can write `... use for C(RUN) build instructions.` to use code formatting for `RUN`.